src/containers/Review/ReviewHome.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Message, Segment, Header } from 'semantic-ui-react'
+import { Loading } from '../../components'
+import strings from '../../utils/constants'
+import useGetFullApplicationStructure from '../../utils/hooks/useGetFullApplicationStructure'
+import { AssignmentDetailsNEW, FullStructure } from '../../utils/types'
+import ReviewSectionAssignment from './ReviewSectionAssignment'
+
+interface ReviewHomeProps {
+  assignments: AssignmentDetailsNEW[]
+  structure: FullStructure
+}
+
+const ReviewHome: React.FC<ReviewHomeProps> = ({ assignments, structure }) => {
+  const { error, fullStructure: fullApplicationStructure } = useGetFullApplicationStructure({
+    structure,
+    firstRunValidation: false,
+    shouldCalculateProgress: false,
+  })
+
+  if (error) return <Message error title={strings.ERROR_APPLICATION_OVERVIEW} list={[error]} />
+
+  if (!fullApplicationStructure) return <Loading />
+
+  return (
+    <>
+      {fullApplicationStructure.sortedSections?.map(({ details: { id, title } }) => (
+        <Segment key={id}>
+          <Header>{title}</Header>
+          {assignments.map((assignment) => (
```

Comment:
To be checked: is this matching with current section? How?

Reply:
so for each section ->
  for each assignment ->
     figure out progress etc.. and render